Zend/zend_fibers.c

```diff
@@ -481,7 +494,6 @@ ZEND_API void zend_fiber_start(zend_fiber *fiber, zval *params, uint32_t param_c
 	fiber->fci.named_params = named_params;
 
 	if (!zend_fiber_init_context(&fiber->context, zend_fiber_execute, EG(fiber_stack_size))) {
-		zend_throw_exception(NULL, "Could not create fiber context", 0);
```

Comment:
This was intentionally throwing an `Exception`, not `Error`, because it does not indicate a programming error but an environment error that is can happen during normal operation (normally, if more than 65k fibers are created).

Though maybe it should use something more specific than just `Exception`.

Reply:
> This was intentionally throwing an `Exception`, not `Error`

Sorry... I know that. It's a typo error... I just copy the code from [my implementation version](https://github.com/twose/php-src/commit/0a43b34692d74c1623a6c0ed0ac93ad310f5d619#diff-749481a6049bac0d06dac9e3dc3ea4c1caf28874ce4c1e2478ea2b1a25672ba3R143) and remove the `zend_ce_fiber_exception` because we have not declared it, I never noticed that I misuse `zend_throw_error` instead of using `zend_throw_exception` to throw an exception so it becomes `Error` here...

> Though maybe it should use something more specific than just Exception.

So should we declare `FiberException` here?